server/src/main/java/com/cloud/storage/StorageManagerImpl.java

```diff
@@ -2331,6 +2336,17 @@ public ImageStore updateImageStoreStatus(Long id, Boolean readonly) {
         return imageStoreVO;
     }
 
+    @Override
+    public void syncHardwareCapability(long poolId) {
+        StoragePoolVO pool = _storagePoolDao.findById(poolId);
+        // find the host
+        List<StoragePoolHostVO> hosts = _storagePoolHostDao.listByPoolId(poolId);
+        if (hosts.size() > 0) {
+            StoragePoolHostVO host = hosts.get(0);
+            _agentMgr.easySend(host.getHostId(), new ModifyStoragePoolCommand(false, pool));
```

Comment:
can use separate cmd for storage capability ?

Reply:
Done